wayback/__init__.py

```diff
@@ -2,4 +2,7 @@
 __version__ = get_versions()['version']
 del get_versions
 
-from ._client import WaybackClient, WaybackSession  # noqa
+from ._client import (  # noqa
+    original_url_for_memento,
```

Comment:
Obviously this is the one I was using, but I’m wondering whether we should expose `memento_url_data` alongside it or instead (since it gets *all* the useful info out of the URL).

Reply:
:+1: for instead.